libmscore/score.cpp

```diff
@@ -436,7 +436,9 @@ void Score::fixTicks()
                   else if (isMaster() && (s->segmentType() == Segment::Type::ChordRest)) {
                         foreach (Element* e, s->annotations()) {
                               if (e->type() == Element::Type::TEMPO_TEXT) {
-                                    const TempoText* tt = static_cast<const TempoText*>(e);
+                                    TempoText* tt = static_cast<TempoText*>(e);
```

Comment:
actually I'm not sure on this, the new style cast won't work when this gets applied to 2.0.4, which might be possible?